src/fonts.css

```diff
@@ -91,26 +85,11 @@ select {
 }
 
 nav {
-  font-size: $font-size-nav-mobile;
+  font-size: $font-size-default;
 }
 
 @media only screen and (min-width: 992px) {
-  .txt-size-1 {
-    font-size: $font-size-1-desktop;
-  }
-  .txt-size-2 {
-    font-size: $font-size-2-desktop;
-  }
-  .txt-size-3 {
-    font-size: $font-size-3-desktop;
-  }
-  .txt-size-h1 {
-    font-size: $font-size-h1-desktop;
-  }
-  button > p {
-    font-size: $font-size-2-desktop;
-  }
-  nav {
-    font-size: $font-size-nav-desktop;
+  html {
+    font-size: 14pt;
```

Comment:
why not variable? Why not rem units?

Reply:
Can be a variable of course. But there will be two new variables for two root values.